### ISSUE TITLE
Make the icons in the side panel configurable and clickable

### ISSUE THREAD

**Issue by adityathakurxd** [adityathakurxd/flutter-portfolio#10]
<img width="336" alt="Screenshot 2023-03-24 at 1 49 03 PM" src="https://user-images.githubusercontent.com/53579386/227463604-922ec33b-c03b-42f7-af47-1ca9108b3b10.png">

The launch icons don't work and can be made configurable and clickable.

**Comment by Vethya** [comment]
Hi, I would like to work on this issue. Can you tell me more on how I should do this?

**Comment by adityathakurxd** [comment]
Sorry for getting back a little late on this one. If you're still interested in working on this issue, here is what I believe should be done:

The arrow icon as on the image in the issue can be made clickable to launch a web URL using an external package.
These URLs can be made customisable for the portfolio owner to specify.

**Comment by Vethya** [comment]
Yes, I'm still interested in working on this issue. I hope you won't mind me taking a bit of time on this issue since I'm quite busy lately, but regardless I will give it a try.

**Comment by adityathakurxd** [comment]
Yes sure, please take your time.
